### PR TITLE
Filter merge test

### PR DESCRIPTION
**Pull request by ArchStanton987**
Dear reviewers,

This is our first version of FoodRoulette. As you'll notice, it features several components at once. It includes "home" (homepage), filter pages for both cocktails and pizza, and the page which displays ingredients.
So far, buttons are not yet functional but we have now a global insight. Ingredients also displays from 3 to 7 cards (randomly, which resets when you refresh the page). So far, ingredients displays doesn't come directly from the API, but in an array stored in app.js.
From this scaled mockup we'll be able to make components alive during our second sprint.

Best regards,

FoodRoullette Dev Team

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
-# reims-0919-js-pudding-arsenic
```
Comment (erabin05) [inline README.md:1]: Personalise the README.md to introduce your projects : quick description of what it is and team members

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+		let idNumbers = Math.floor(Math.random() * 5)
+		return numbers[idNumbers]
+	}
+	render() {
+		const ingredientNumber = this.randomIngredientNumber()
+		let ingredientArray = []
+		const showIngredientCall = <ShowIngredients {...product[2]} />
+		for (let i = 0; i < ingredientNumber; i++) {
+			ingredientArray.push(showIngredientCall)
+		}
```
Comment (erabin05) [inline src/App.js]: render is not the place to put all this code.
Some can be in a state, some in methods or directly in JSX

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+		console.log(ingredientArray)
```
Comment (erabin05) [inline src/App.js]: Remove console.log() for PR

File: src/components/Home.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import React from "react"
+import Navbar from "./Navbar"
+import CocktailFilters from "./CocktailFilters"
+import "./Home.css"
+
+class Home extends React.Component {
```
Comment (erabin05) [inline src/components/Home.js]: If the Component is stateless it must be a function

File: src/components/Home.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+								className="imageCocktail"
+								src="https://zupimages.net/up/19/43/4p0w.jpg"
+								alt="cocktail"
+							></img>
+						</figure>
+
+						<button type="button" onClick={CocktailFilters}>
+							Get a cocktail
+						</button>
+						{/* <a href="#CocktailFilters">Get a cocktail</a> */}
```
Comment (erabin05) [inline src/components/Home.js]: No dead code in PR

File: src/components/HomeText.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+//import React from "react";
+
+const HomeText = () => {}
+
+//export default HomeText
```
Comment (erabin05) [inline src/components/HomeText.js]: Don't need this empty component in PR

File: src/components/ShowIngredients/ShowIngredients.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import "./ShowIngredients.css"
+
+class ShowIngredients extends React.Component {
```
Comment (erabin05) [inline src/components/ShowIngredients/ShowIngredients.js:4]: Can be a function because stateless

**Review by erabin05 — CHANGES_REQUESTED** [review]
Some changes to do : see comments to know what.
Next time try to have less files in your PR (30 files :'( )

File: src/components/ShowIngredients/ShowIngredients.js (hunk 7 above)
Comment (ghost) [reply src/components/ShowIngredients/ShowIngredients.js:4]: Is now a function.

File: src/components/HomeText.js (hunk 6 above)
Comment (ghost) [reply src/components/HomeText.js]: Removed.

File: src/components/Home.js (hunk 5 above)
Comment (ghost) [reply src/components/Home.js]: Removed.

File: src/components/Home.js (hunk 4 above)
Comment (ghost) [reply src/components/Home.js]: Is now a function.

File: src/App.js (hunk 3 above)
Comment (ghost) [reply src/App.js]: Removed.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:1]: Done.

File: src/App.js (hunk 2 above)
Comment (ArchStanton987) [reply src/App.js]: This piece of code has been modified : it is now in a component "ShowIngredientList", which calls "ShowIngredients" and is called by "App".

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**